runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -115,6 +116,9 @@
 		13113, "Larran's small chest"
 	);
 
+	// Last man standing map regions
+	private static final int[] LAST_MAN_STANDING_REGIONS = {13658, 13659, 13914, 13915, 13916};
```

Comment:
Should this be an immutable set? Reference:
https://github.com/runelite/runelite/pull/1286#discussion_r179928302

Reply:
Good call, should be good now